### DISCUSSION TITLE
get final url

### DISCUSSION THREAD

**Discussion by jimmywarting** [axios/axios#4571] [Q&A] [answered]
like with the fetch api you can get the final redirected url by looking at the `response.url` 
it might not always be the same as the requested url... do axios have anything similar?

**Comment by carlos-dubon** [comment, marked as answer]
You can access it by accessing response.config.url or response.request.responseURL, though if you only want the URI you would need to use regex or you could use JS by utilizing a array or string method.

Example:

```JavaScript
const axios require('axios');
const url = require('url');

axios(req)
  .then(response => {
    const parsedURL = url.parse(response.config.url);
    console.log(parsedURL.host);
  })
  .catch(err => console.error(err));
```
